insights/tests/test_syslog.py

```diff
@@ -11,6 +11,12 @@
 Apr 22 10:37:32 boy-bona crontab[28951]: (root) LIST (root)
 Apr 22 10:40:01 boy-bona CROND[30677]: (root) CMD (/usr/lib64/sa/sa1 -S DISK 1 1)
 Apr 22 10:41:13 boy-bona crontab[32515]: (root) LIST (root)
+Jul 10 15:08:35 shelly audit[1]: USER_AVC pid=1 uid=0 auid=4112341244 ses=49287573824 subj=system_u:system_r:init_t:s0 msg='avc:  received policyload notice (seqno=2)
+                                 exe="/usr/lib/systemd/systemd" sauid=0 hostname=? addr=? terminal=?'
+Jul 10 15:08:35 shelly audit[1]: USER_AVC pid=1 uid=0 auid=4112341244 ses=49287573824 subj=system_u:system_r:init_t:s0
+                                 msg='avc:  received policyload notice (seqno=2)
+                                 exe="/usr/lib/systemd/systemd" sauid=0 hostname=? addr=? terminal=?'
+Jul 10 15:08:35 shelly systemd[1]: Reloading.
```

Comment:
@psav are messages actually stored in the file this way or is this an artifact of how you view the messages?  Insights core captures an unfiltered file like this:
```
with open(path, "rU") as f:
    results = [l.rstrip("\n") for l in f]
return results
```
Would you get the same results with continued lines if the file is read in this manner?